lake/utils/util.py

```diff
@@ -187,6 +219,52 @@ def extract_formal_annotation(generator, filepath, module_attr="agg"):
         print(sram_tb_shared, file=open("sram_tb_shared.txt", "w+"))
 
 
+def extract_formal_annotation_colat(generator, filepath, mem_names, edges):
```

Comment:
Huge nitpick, but shouldn't it be `collat`?

Reply:
Lol yes, fixed thanks